python/skupper_router/management/skrouter.json

```diff
@@ -1184,6 +1184,12 @@
                     "default": "0.0.0.0",
                     "create": true
                 },
+                "backlog": {
+                    "description": "Sets the value of the backlog parameter in call to listen(). This controls the maximum queue depth the OS will use for some stages of TCP connection creation.  Value should be integer >= 1, or 'max'. If 'max', the value will be set to the upper limit allowed by the OS.'",
+                    "type": "string",
```

Comment:
It might be easier to make the type "integer" and use qd_entity_opt_long(e, "backlog", 0) and treat the zero value as MAXCONN.

In today's brave new world of skupper humans no longer get to access to the config file directly - it's all through the skupper cli.  Having user friendly names like "max" aren't really necessary.

The other advantage is using and integer will fix the leak errors you're seeing in CI.